site/source/tutorials/reactive_programming.md

```diff
@@ -0,0 +1,58 @@
+# Reactive Programming
+
+Reactive programming is concenered with handling change, specifically, the _propagation of change_. As web applications have become more and more asynchronous, keeping track of application state has become increasingly complicated. With traditional programming patterns we rely on event listeners or try to keep track of application state manually.
```

Comment:
patterns we rely on event listeners or try to keep track of application state manually -> patterns, we traditionally rely on event listeners or try manually track application state